incremental/ubuntu16-upgrade-to-rocky.sh

```diff
@@ -28,6 +28,7 @@ export RPC_ANSIBLE_VERSION="2.5.5"
 
 echo "Starting Queens to Rocky Upgrade..."
 
+check_for_osa_bootstrap
```

Comment:
Minor nit: Rename to ensure_osa_bootstrap, since it's not just checking but also bootstrapping if needed.

Reply:
Good call, changes pushed.